addon/mixins/ajax-request.js

```diff
@@ -42,16 +42,13 @@ const JSONAPIContentType = 'application/vnd.api+json';
 // Header names are case-insensitive
 // https://www.w3.org/Protocols/rfc2616/rfc2616.html
 function retrieveHeader(headers, name) {
-  let found = null;
-  
+  var found = null;
```

Comment:
You are right. I thought `var` was needed to set the value from within a child-scope (since block vs. function scoping is the difference between the two), but reading up on the docs I think `let` will work, i.e. it's still available within child-blocks, only the shadowing is different between the two.